### PR TITLE
chore: Update Code of Conduct and Contributing guide

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Fixes a bunch of markdown structure issues in our guides

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Test locally on your own infrastructure
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation)) 
- [ ] Update or add tests 🧪
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
-## Community Code of Conduct
-cloudquery adopts the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+# Community Code of Conduct
```
Comment (erezrokah) [inline CODE_OF_CONDUCT.md:1]: Should start with an H1 header

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -5,87 +5,54 @@
 
 The following is a set of guidelines for contributing to this repository.
 
-
-#### Table of Contents
```
Comment (erezrokah) [inline CONTRIBUTING.md:9]: GitHub automatically generates table of contents for markdown files

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -5,87 +5,54 @@
...
-
-[How Can I Contribute?](#how-can-i-contribute)
-  * [Reporting Bugs and Suggesting Enhancements](#reporting-bugs-and-suggesting-enhancements)
-  * [Your First Code Contribution](#your-first-code-contribution)
-
 ## Code of Conduct
 
-This project and everyone participating in it is governed by the [CloudQuery Code of Conduct](https://github.com/cloudquery/cloudquery/blob/main/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. To report inappropriate behavior in violation of the code, please start by reaching out to us on our [Discord channel](https://cloudquery.io/discord).
-
+This project and everyone participating in it is governed by the [CloudQuery Code of Conduct](./CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. To report inappropriate behavior in violation of the code, please start by reaching out to us on our [Discord channel](https://cloudquery.io/discord).
```
Comment (erezrokah) [inline CONTRIBUTING.md:10]: Better to use relative links as they work on PRs

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -5,92 +5,70 @@
...
-## How Can I Contribute?
+* Interacting with CloudQuery CLI for initialization and configuration
+* Helper functions for defining table schemas
+* Methods for testing the resource
+* Framework for running and building a plugin locally
 
-### Reporting Bugs and Suggesting Enhancements
+#### CloudQuery Plugins
 
-See [issue_reporting.md](issue_reporting.md) for more details on how to do this.
```
Comment (erezrokah) [inline CONTRIBUTING.md:70]: Probably more familiar to keep the information on how to submit issues in the issues templates

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -5,92 +5,70 @@
...
+* Interaction with remote data sources:
+  * Initialization of clients
+  * Authentication
+  * Fetching of configuration information
+* More information about developing your own plugin can be found [here](https://cloudquery.io/docs/developers/developing-new-provider)
 
-### Your First Code Contribution
+## How Can I Contribute?
 
-Unsure where to begin contributing to CloudQuery? You can start by looking through these `beginner` and `help-wanted` issues:
```
Comment (erezrokah) [inline CONTRIBUTING.md:74]: `good first issue` labels are default in GitHub repositories and seem more common to use

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**